mods/d2k/weapons.yaml

```diff
@@ -9,74 +9,95 @@ LMG:
 		ContrailDelay: 0
 		ContrailUsePlayerColor: true
 	Warhead@1Dam: SpreadDamage
-		Spread: 96
-		Damage: 15
+		Spread: 48
+		Falloff: 100, 100, 65, 30, 0, 0, 0
```

Comment:
You don't need to define 0 multiple times, the last two can simply be removed (applies to all other weapons, too).